### PR TITLE
Add retrospective eighth birthday post

### PR DESCRIPTION
**Pull request by bitsondatadev**
- Adding a library to embed tweets.
 - Adding retrospective eighth birthday post.

### CODE REVIEW

**Comment by bitsondatadev** [comment]
@mosabua Adding the retrospective eighth birthday post. I am not sure what happened to my cla signature. It may have become invalidated after I changed my github name.

**Comment by findepi** [comment]
@cla-bot check

**Comment by findepi** [comment]
@bitsondatadev please update author & committer email in commits or add the email address to your profile (if using a new email)
-- github does not recognize the commits as yours yet

**Comment by bitsondatadev** [comment]
Thanks @findepi !

File: _config.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ presto_version: 340
 
 google_analytics: UA-133457846-1
 
-slack_url: "https://join.slack.com/t/prestosql/shared_invite/enQtNTMyNzU2NzQ1NzQ4LWUyMjY4NTA0NWU0ZDlkMjM4Y2JhODdiZWZiZWQ2NDNhZDVhOGY5ZTk4NDY0OGIzOTE2YzI3YjhlMmUyNDJiNGM"
+slack_url: "https://join.slack.com/t/prestosql/shared_invite/zt-h88upx5f-gVb8XUu16W_jo4I2Cw8ijA"
```
Comment (mosabua) [inline _config.yml:5]: Why this change?

File: _posts/2020-08-08-presto-eighth-birthday.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+title:  "Happy Eighth Birthday Presto!"
+author: Brian Olsen
+excerpt_separator: <!--more-->
+---
+
+Today, Presto turned eight years old! As Presto co-creator
+Dain Sundstrom points out, there’s a reason why the eighth birthday is a
+little special:
+
+https://twitter.com/daindumb/status/1292296395219595264
```
Comment (mosabua) [inline _posts/2020-08-08-presto-eighth-birthday.md:12]: I would just use an image and get rid of the jekyll plugin for now .. unless we plan to do lots more twitter embedding.

File: _config.yml (hunk 1 above)
Comment (bitsondatadev) [reply _config.yml:5]: I'm not sure why this is showing in the diff. This is the line that is currently on master. https://github.com/prestosql/prestosql.io/blob/master/_config.yml#L5
This change was merged in from master but shouldn't be showing as one of the files changed.

File: _posts/2020-08-08-presto-eighth-birthday.md (hunk 2 above)
Comment (bitsondatadev) [reply _posts/2020-08-08-presto-eighth-birthday.md:12]: I think I am planning on it. Typically with announcements or promoting individuals within the community, I like linking individual's tweets about their work, as well as, linking their GitHub accounts.

File: _config.yml (hunk 1 above)
Comment (findepi) [reply _config.yml]: > This change was merged in from master but shouldn't be showing as one of the files changed.

oh, perhaps because you used `git merge`. We do not merge, please `git rebase -i master` instead

Comment (bitsondatadev) [reply _config.yml]: Just needed to merge in upstream master.

Comment (bitsondatadev) [reply _config.yml]: okay, removed the extra commit and squashed some extra changes Martin asked me to make.

**Comment by bitsondatadev** [comment]
@martint do you have any feedback for this post? Otherwise, I can ask @mosabua to do a quick review.
